src/itemLogin/ItemLoginAuthorization.tsx

```diff
@@ -91,7 +91,7 @@ const ItemLoginAuthorization =
       // the item could be fetched without errors
       // because the user is signed in and has access
       // or because the item is public
-      if (item && !item.isEmpty()) {
+      if (item && !item.toSeq().isEmpty()) {
```

Comment:
I think here there's no case for an empty item.. probably you can remove the second condition?